blocks/subscriptions-block/components/Cart/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useFusionContext } from "fusion:context";
+import getProperties from "fusion:properties";
+import { usePhrases, Heading, Link } from "@wpmedia/arc-themes-components";
+import useSales from "../useSales";
+import currency from "../../utils/currency";
+import Item from "./item";
+
+const getPaymentInfo = ({ origin, endpoint, priceCode }) =>
+	fetch(`${origin}${endpoint}${priceCode}/1/${new Date().getTime()}`, {}).then((res) => res.json());
```

Comment:
What's the purpose of `new Date().getTime()` in the fetch URL? Is it to bypass cache? If it is, is the delivery team aware of this and ok with it?

Reply:
@vgalatro I checked with our BE team, ` /retail/public/v1/pricing/paymentInfo/${priceCode}/${retailCycle}/${time}` provides applicable Rate info of the provided price for provided retailCycle (In this case 1) at given $time. Time is a required attribute to be passed to API.

Reply:
Okay, just wanted to make sure it was something necessary.

Reply:
@vgalatro thx for this feedback.
This is one of our oldest APIs, and I am sure our API standards / DEL's recommended approach for optimizing our APIs has changed.

@LauraPinilla I think it is worth investigating the overall cost of running this API, and see if there are opportunities for optimizing & reducing our AWS costs by figuring out a different way to call / return this info.

Since the smallest time resolution that our clients can use is `daily` for price rates / renewals, maybe for now, instead of calling this API with `new Date().getTime()`, can we use the most recent past midnight for the user's locale?